### DISCUSSION TITLE
Using pick to fetch deeper levels of arrays and objects

### DISCUSSION THREAD

**Discussion by dannyvalle** [nuxt/nuxt#17725] [Questions] [answered]
I have a data structure (see below) in a "products.ts" file in my server/api directory that is more than one level deep. How can I use the "pick" feature with the "useFetch" method to grab for example, just "category-2"?

`
let products = {
	  "name" : "Page Title",
	  "description" : "Description here",
	  "products" : {
		"category-1" : {
				"name" : "Category Name",
				"description" : "Description here",
				"products" : [
					  {
							"name" : "Product Name"
					  },
					  {
							"name" : "Product Name"
					  }
				]
		  },
		  "category-2" : {
				"name" : "Category Name",
				"description" : "Description here",
				"products" : [
					  {
							"name" : "Product Name"
					  },
					  {
							"name" : "Product Name"
					  }
				]
		  },
		  "category-3" : {
				"name" : "Category Name",
				"description" : "Description here",
				"products" : [
					  {
							"name" : "Product Name"
					  },
					  {
							"name" : "Product Name"
					  }
				]
		  }
	  }
}
export default () => {
	  return products
}
`

**Comment by stafyniaksacha** [comment, marked as answer]
Hi! This might help: https://github.com/nuxt/framework/discussions/1407
